### PR TITLE
[Merged by Bors] - [Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/InnerProductSpace/l2Space.lean [hunk 1]
```diff
@@ -240,7 +240,6 @@ protected theorem linearIsometry_apply_single {i : ι} (x : G i) :
   · simp [h]
 #align orthogonal_family.linear_isometry_apply_single OrthogonalFamily.linearIsometry_apply_single
 
-@[simp]
```
Comment (ocfnash) [inline Mathlib/Analysis/InnerProductSpace/l2Space.lean:243]: Why drop this `simp`?

**Review by ocfnash — APPROVED** [review]
Thanks!

One question but otherwise LGTM

bors d+

File: Mathlib/Analysis/InnerProductSpace/l2Space.lean (hunk 1 above)
Comment (Ruben-VandeVelde) [reply Mathlib/Analysis/InnerProductSpace/l2Space.lean:243]: `simp` proves it, and `#lint` fails on the `simpNF` linter. I don't know why CI doesn't catch it.

**Comment by Ruben-VandeVelde** [comment]
bors merge

**Pull request closed without merging**